### PR TITLE
config: list several modules that were missing

### PR DESCRIPTION
**Pull request by czarkoff**
Add missing modules to configuration file generator.  I did not add deprecated and obscure modules.

(*Referencing PR #173 because it would make sense to merge this before release.*)

### CODE REVIEW

File: src/config.c [hunk 1]
```diff
@@ -651,9 +656,14 @@ int config_write_template(const char *file, const struct config *cfg)
 	(void)re_fprintf(f, "#module\t\t\t" MOD_PRE "avcodec" MOD_EXT "\n");
 #endif
 	(void)re_fprintf(f, "#module\t\t\t" MOD_PRE "vp8" MOD_EXT "\n");
+	(void)re_fprintf(f, "#module\t\t\t" MOD_PRE "vp9" MOD_EXT "\n");
+	(void)re_fprintf(f, "#module\t\t\t" MOD_PRE "h265" MOD_EXT "\n");
+	(void)re_fprintf(f, "#module\t\t\t" MOD_PRE "daala" MOD_EXT "\n");
```
Comment (alfredh) [inline src/config.c]: daala is a very experimental codec, and it is now deprecated in favour of "AV1" codec.
I dont think we should include it in the default config

File: src/config.c [hunk 2]
```diff
@@ -715,6 +735,9 @@ int config_write_template(const char *file, const struct config *cfg)
 	(void)re_fprintf(f, "module_app\t\t" MOD_PRE "auloop"MOD_EXT"\n");
 	(void)re_fprintf(f, "module_app\t\t"  MOD_PRE "contact"MOD_EXT"\n");
 	(void)re_fprintf(f, "module_app\t\t"  MOD_PRE "debug_cmd"MOD_EXT"\n");
+	(void)re_fprintf(f, "module_app\t\t"  MOD_PRE "dtmfio"MOD_EXT"\n");
+	(void)re_fprintf(f, "module_app\t\t"  MOD_PRE "echo"MOD_EXT"\n");
```
Comment (alfredh) [inline src/config.c]: the echo module is not compiled by default, perhaps it should be commented out ?

**Comment by alfredh** [comment]
many thanks for your patch, it was very useful to clean up the config


I tested with your default config, and get a couple of warnings:

```
dl: mod: ./dtmfio.so (dlopen(./dtmfio.so, 6): image not found)
module dtmfio.so: No such file or directory
dl: mod: ./echo.so (dlopen(./echo.so, 6): image not found)
module echo.so: No such file or directory
```


these modules are not built by default, so I think they should be commented out
in default config.

**Comment by czarkoff** [comment]
From my reading of `mk/modules.mk` module `dtmfio` is enabled by default on Linux and disabled elsewhere.  Thus I've wrapped it in `#ifdef LINUX`.  Is it fine?

**Comment by alfredh** [comment]
thank you @czarkoff  :)

**Pull request merged**